File: integration-tests/gradle/src/main/resources/add-remove-extension-single-module-kts/src/main/java/org/acme/ExampleResource.java
package org.acme;

import javax.inject.Inject;
import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/hello")
public class ExampleResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "Hello from Test";
    }
}

File: integration-tests/gradle/src/test/resources/kotlin-grpc-project/src/main/kotlin/org/acme/ExampleResource.kt
package org.acme

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType

import io.quarkus.example.HelloMsg

@Path("/hello")
class ExampleResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello() = "hello" + HelloMsg.Status.TEST_ONE.getNumber()
}